app/helpers/application_helper.rb

```diff
@@ -200,22 +200,14 @@ def popover_for_user_name(user,
 
   # No more dirty hack
   def user_avatar_absolute_url(user, style)
-    begin
-      unless missing_avatar(user, style)
-        image = File.open(user.avatar_variant(style))
-        encoded_data = Base64.strict_encode64(image)
-        avatar_base64 = "data:#{user.avatar_content_type};base64,#{encoded_data}"
-        return avatar_base64
-      end
-    rescue StandardError => e
-      Rails.logger.error e.message
+    avatar_link = user.avatar_variant(style)
```

Comment:
Doesn't avatar_variant check if avatar is attached?

Reply:
`avatar_variant` return variant or link to avatar placeholder. And if it return variant we need obtain `service_url`, but for avatar we can use link as is.